web/src/engine/websites/MangaHubRU.ts

```diff
@@ -0,0 +1,49 @@
+import { Tags } from '../Tags';
+import icon from './MangaHubRU.webp';
+import { Chapter, DecoratableMangaScraper, Page, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from './../FetchProvider';
+
+type APIPages = {
+    scans: {id : number, src : string}[]
+}
+
+@Common.MangaCSS(/^https?:\/\/mangahub\.ru\/title\//, '#title-detail div.col div.card-padding-x h1')
+@Common.MangasMultiPageCSS('/explore?page={page}', 'div.container div.card a.comic-grid-name')
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangahubru', `MangaHub.ru`, 'https://mangahub.ru', Tags.Language.Russian, Tags.Media.Manga, Tags.Media.Manhua, Tags.Media.Manhwa, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const url = new URL(manga.Identifier.replace('/title/', '/chapters/'), this.URI);
+        const request = new FetchRequest(url.href);
+        const data = await FetchCSS<HTMLAnchorElement>(request, 'div.detail-chapters a.d-inline-flex');
+        return data.map(element => {
+            const span1 = element.querySelector('span.text-truncate'); //main title to trim
+            const span2 = span1.querySelector('span');//secondary title
+            let title = '';
+            if (span2) {
+                span1.removeChild(span2);
+                title = span1.textContent.trim() + span2.textContent.trim();
+            } else title = span1.textContent.trim();
```

Comment:
Try to always use curly brackets for block statements to comply with clean code guidelines, it is likely that the corresponding lint rule for this will be activated in the future...